integration-test/src/test/java/org/ehcache/integration/statistics/TierCalculationTest.java

```diff
@@ -67,7 +68,7 @@ public void before() throws Exception {
         .withExpiry(ExpiryPolicyBuilder.timeToLiveExpiration(Duration.ofMillis(TIME_TO_EXPIRATION)))
         .build();
 
-    StatisticsService statisticsService = new DefaultStatisticsService();
+    StatisticsService statisticsService = new DefaultStatisticsService(new DefaultStatisticsServiceConfiguration());
```

Comment:
Wouldn't have a default constructor for `DefaultStatisticsService` creating a `DefaultStaticticsServiceConfiguration` make sense?